tests/integration/questionnaire/__init__.py

```diff
@@ -0,0 +1,6 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+def add_person(test_case: IntegrationTestCase, first_name: str, last_name: str):
+    test_case.post({"anyone-else": "Yes"})
+    test_case.post({"first-name": first_name, "last-name": last_name})
```

Comment:
This can't be a generic IntegrationTestCase method as it relies on ids that can vary between schemas.

Reply:
Are you suggesting removing the type hint (which is valid, even if the ids do change) or making this a method on a sub-class of IntegrationTestCase?

Reply:
The issue I was highlighting is that the ids being posted will only exist on certain schemas, so shouldn't be in `IntegrationTestCase` as methods in there should work for all schemas. A sub-class would be ok.

Reply:
All done in b2d3f291f